js/ui/backgroundManager.js

```diff
@@ -4,7 +4,7 @@ const Gio = imports.gi.Gio;
 const Lang = imports.lang;
 const Mainloop = imports.mainloop;
 
-class BackgroundManager {
+var BackgroundManager = class {
```

Comment:
Can you give the reason why this is important? Dynamic reloading / overwrite at runtime?